x-pack/plugins/dashboard_enhanced/public/services/drilldowns/actions/flyout_edit_drilldown/flyout_edit_drilldown.test.tsx

```diff
@@ -37,6 +38,9 @@ function createAction(overlays = coreMock.createStart().overlays) {
   };
   return new FlyoutEditDrilldownAction(params);
 }
+const dynamicActionsState$ = new BehaviorSubject<DynamicActionsSerializedState['enhancements']>({
+  dynamicActions: { events: [{} as any] },
```

Comment:
Can events just be `[]` to avoid the `any` cast?

Reply:
For this one, we want at least one event by default - so I've switched the casting to `SerializedEvent` in 343e464a979251cae24177f87eb348c52ab673b0 rather than `any` as you suggested for the other tests :+1: